### DISCUSSION TITLE
40 rahip sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#101] [Zeka Sorusu]
"dış dünyaya kapalı bi manastırda sessizlik yemini etmiş 100 tane rahip yaşamaktaymış. bu 100 rahibin 40 tanesinin saf olmayan düşüncelerinden ötürü alınlarında siyah bir leke oluşmuş. bi akşam yemeği toplantısında baş rahip ayağa kalkıp:
"kardeşlerim, mecburi olarak sessizlik yeminimi bozuyorum. fark etmişsinizdir, aramızda bazılarının alınlarında günah lekeleri var. bu arkadaşlar -ben de aralarında olabilirim- lütfen durumlarını anladıklarında ilk fırsatta intihar etsinler."
der ve tekrar konuşma orucuna geri döner.
baş rahibin konuştuğu gece 1. gecedir.
40. gece alnı kara olan 40 rahip alınlarının kara olduğunu anlayıp intihar ediyorlar. alnı ak rahiplerin hiçbiri intihar etmiyor.
soru: bu nasıl olmuştur? alnı karalar durumu nasıl anlamıştır?
önemli hatırlatma: bu bir matematik sorusudur ve "rahipler aynaya bakar" (ayna yok manastırda) ya da "birbirlerine kaş göz işareti yaparlar" (iletişim yok, yalnızca konuşma değil yasak olan) gibi sivri zeka cevaplar geçersizdir. vereceğiniz cevap olabildiğince soyut akıl yürütme ürünü olmalıdır. kelime oyunu falan yok. ne anlıyosanız o.
koşullar:

bütün rahipler soyut akıl yürütme konusunda mükemmeller ve bu durum da bütün rahipler tarafından biliniyor. hatta, bu durumun bütün rahipler tarafından bilindiği de bütün rahipler tarafından biliniyor.

bütün rahipler her akşam yemekte birbirlerinin yüzünü görerek bir diğerinin alnının kara olup olmadığını görebiliyor. gün içinde hepsi kendi mıntıkalarında. bunun hikmeti şu: günde 1 kere bilgi güncellemesi oluyor.

bir rahibin kendi alnının karalığını ya da aklığını anlamasının tek yolu diğer rahipleri gözlemleme (bi ipucu daha veriyim, bi diğer rahibe bakınca tek görülen alnının kara olup olmadığı ve hala hayatta olup olmadığı. yemeğe gelmişse yaşıyo, sabah hücresinden cesedini çıkardıysak yaşamıyo demektir, akşam da yemeğe gelemez 🙂 ve soyut akıl yürütme.

**Comment by DelphiLover** [comment]
Cevap gayet basit. Günahkar rahipler başkalarının kendilerine söylemesine veya aynaya bakmaya gerek kalmadan intihar etmişlerdir. Çünkü kendilerinin günah işlediğini kendileri bilmektedirler.. Günah işleyenler intihar etmişlerdir.
